Jetcaster/gradle/libs.versions.toml

```diff
@@ -17,7 +17,7 @@ androidx-core-splashscreen = "1.0.1"
 androidx-corektx = "1.13.1"
 androidx-glance = "1.1.0"
 androidx-lifecycle = "2.8.2"
-androidx-navigation = "2.7.7"
+androidx-navigation = "2.8.0-beta03"
```

Comment:
The libs.versions.toml is controlled by the main top level one, and propogated to each sample. Please make changes to that instead of invidually.

Reply:
I'll keep this as a draft until the Navigation version stabilizes at 2.8.0. 🥲